tensorbay/client/cli.py

```diff
@@ -368,7 +368,7 @@ def _echo_data(dataset_name: str, segment_name: str, data_iter: Iterable[str]) -
 
 
 def _ls_dataset(gas: GAS, info: TBRN, list_all_files: bool) -> None:
-    dataset = gas._get_dataset(info.dataset_name)  # pylint: disable=protected-access
+    dataset = gas.get_dataset(info.dataset_name)
```

Comment:
old `_get_dataset` method can get `Dataset` and `FusionDataset` without checking
After your modification, the CLI `gas ls` lost the ability to get `FusionDataset`